components/Home/UpcomingEvents.tsx

```diff
@@ -1,43 +1,48 @@
 import Link from 'next/link';
-import { FC } from 'react';
+import { FC, useContext } from 'react';
 import { Button, Card, Col, Container, Row } from 'react-bootstrap';
 
+import { I18nContext } from '../../models/Translation';
 import { ArticleMeta } from '../../pages/api/core';
 import { SectionTitle } from './SectionTitle';
 
 interface UpcomingEventsProps {
   events: ArticleMeta[];
 }
 
-export const UpcomingEvents: FC<UpcomingEventsProps> = ({ events }) => (
-  <div className="py-5 bg-white w-100 m-0">
-    <Container>
-      <SectionTitle>近期活动</SectionTitle>
+export const UpcomingEvents: FC<UpcomingEventsProps> = ({ events }) => {
+  const { t } = useContext(I18nContext);
 
-      <Row className="g-4" xs={1} sm={2} md={3}>
-        {events.map(({ name, meta, path }) => (
-          <Col key={name}>
-            <Card body>
-              <Card.Title className="text-dark">{name}</Card.Title>
-              <Card.Text className="text-dark">
-                时间: {meta?.start || 'N/A'}
-              </Card.Text>
-              <Card.Text className="text-dark">
-                地点: {meta?.address || 'N/A'}
-              </Card.Text>
+  return (
+    <div className="py-5 bg-white w-100 m-0">
```

Comment:
```suggestion
    <div className="py-5 bg-white">
```
同时，这个布局框应该放到外面引用的地方来决定如何布局，而不是在组件内部写死。